cmd/main.go

```diff
@@ -21,10 +21,10 @@ import (
 	"log"
 	"os"
 	"path/filepath"
-	"strings"
 
 	"github.com/spf13/cobra"
 
+	toolsUtil "sigs.k8s.io/controller-tools/pkg/crd/util"
```

Comment:
we don't generally use camelCase package names (the general practice is for all-lower-case package names, which makes them a bit easier to distinguish when reading code).

Reply:
Thanks for the review, I've changed it to `toolsUtil `